src/connectors/joox.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const playerSelector = '.kIpMXu';
```

Comment:
Could you also use strings directly? For example:
```js
Connector.trackSelector = '.jlfeQT';
```

Reply:
I'd prefer to keep it this way because it's easier to change along with their client-side code.

Reply:
I don't know why this way is easier, but the whole PR looks acceptable for me, so I resolve this conversation.